requirements.txt

```diff
@@ -0,0 +1,29 @@
+# Freeze these depndencies
+Werkzeug==1.0.1
+
+# Dependencies
+Flask==1.1.2
+Flask-API==1.1
+# Flask-Log-Request-ID==0.10.1
+SQLAlchemy==1.3.23
+Flask-SQLAlchemy==2.4.4
+python-dotenv==0.10.3
+psycopg2-binary==2.8.4
+
+# Runtime
+gunicorn==20.0.4
+honcho>=1.0.1
+
+# Code quality
+pylint==2.4.4
+flake8==3.7.9
+
+# Testing
+factory-boy==2.12.0
```

Comment:
usually i would add the dependencies as needed (ie when testing is implemented). this way we can be more confident that each is actually used by the project